README.md

```diff
@@ -1 +1,63 @@
-# my-portfolio-review-2
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Dyary's Portfolio
+
+> this is my portfolio website. It includes info about me, how to contact me and my achivemets and creations as a web developer. It's created with CSS and HTML and used VS Code as my code editor. the fonts I used for this website is Poppins and this is for the whole website. I used Flexbox to style my nav items and social media links. I used media queries to make it responsive. so i works well on mobile devices.
+
+![screenshot](https://i.imgur.com/vWXwTaR.png)
+
+look thorugh the website to see my work and contact me if need.
+
```

Comment:
Consider adding a live demo section to your page.  You can use GitHub pages for that, which can be found in your repository's settings.